styles.css

```diff
@@ -0,0 +1,129 @@
+body {
+  margin: 0;
+}
+
+.wrapper {
+  margin: auto;
+  width: 80%;
+}
+
+.main-content__title {
+  font-family: 'PT Serif', serif;
+}
+
+.logo-name {
+  font-family: 'PT Serif', serif;
+}
+
+.main-content__description {
+  font-family: 'Roboto', sans-serif;
+}
+
+.menu {
+  font-family: 'Roboto', sans-serif;
+  padding: 47px 0;
+}
+
+.main-content__button {
+  font-family: 'Roboto', sans-serif;
+}
+
+.logo, .menu {
+  display: inline-block;
+}
+
+.logo {
+  position: relative;
+  padding: 40px 0;
+  padding-left: 44px;
+  font: bold 24px/31px 'PT Serif', Arial, Helvetica, sans-serif;
+}
+
+.logo__circle {
+  position: absolute;
+  left: 0;
+  top: 50%;
+  transform: translateY(-50%);
+  width: 28px;
+  height: 28px;
+  border-radius: 50%;
+  opacity: .7;
+}
+
+.logo__circle_color_dark-blue {
+  background: #0056EC;
```

Comment:
Codacy found an issue: [Expected "#0056EC" to be "#0056ec" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/kastykina_app.creation/pullRequest?prid=6252582)